sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+SRC_URI="https://github.com/facebookincubator/oomd/releases/tag/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
```

Comment:
Please pop this variable above *DEPEND, in with LICENSE/SLOT. See skel.ebuild for the rough structure/order, although there's too much whitespace in there.